spec/ddtrace/contrib/rails/rack_spec.rb

```diff
@@ -555,6 +591,10 @@ def internal_server_error
       expect(controller_span).to have_error_type('ActionView::Template::Error')
       expect(controller_span).to have_error_stack
       expect(controller_span).to have_error_message
```

Comment:
⬆️  I'm curious: is it possible for a span to `have_error` but not to `have_error_type`? Could we clean up a few of these assertions that get duplicated by other assertions?

This file is already really complex and it seems to keep on growing and accumulating assertions 😅